src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -681,28 +681,20 @@
      * </pre>
      * <p>parses as:</p>
      * <pre>
-     * METHOD_DEF -&gt; METHOD_DEF
-     *     |--MODIFIERS -&gt; MODIFIERS
-     *     |--TYPE -&gt; TYPE
-     *     |   `--LITERAL_VOID -&gt; void
-     *     |--IDENT -&gt; foo
-     *     |--LPAREN -&gt; (
-     *     |--PARAMETERS -&gt; PARAMETERS
-     *     |   |--PARAMETER_DEF -&gt; PARAMETER_DEF
-     *     |   |   |--MODIFIERS -&gt; MODIFIERS
-     *     |   |   |--TYPE -&gt; TYPE
-     *     |   |   |   `--LITERAL_INT -&gt; int
-     *     |   |   `--IDENT -&gt; firstParameter
-     *     |   |--COMMA -&gt; ,
-     *     |   `--PARAMETER_DEF -&gt; PARAMETER_DEF
-     *     |       |--MODIFIERS -&gt; MODIFIERS
-     *     |       |--TYPE -&gt; TYPE
-     *     |       |   `--LITERAL_INT -&gt; int
-     *     |       |--ELLIPSIS -&gt; ...
-     *     |       `--IDENT -&gt; secondParameter
-     *     |--RPAREN -&gt; )
-     *      `--SLIST -&gt; {
-     *          `--RCURLY -&gt; }
+     * |--PARAMETERS -> PARAMETERS
```

Comment:
we need whole block of code to show AST in context of other nodes.
please keep METHOD_DEF

Reply:
Okay sure I will keep the method definition also

Reply:
Hi @romani I am getting merge conflicts while I was trying to pull and push my changed code
I think some other developer is working on it. Please tell me what am I supposed to do.

Reply:
Please read attentively https://github.com/checkstyle/checkstyle/wiki/To-student-of-GSoC

Some links show how to rebase code over latest